.github/workflows/pullrequest.yml.template

```diff
@@ -0,0 +1,76 @@
+# Name of the PlaceHolder                 -  Description
+########################################################################################################################
+# <Name of the WorkFlow>                  - Workflow name. example: Bikes
+# <Path to the service>                   - Path to the service where the developer would like to trigger the Flow/ action on pull request - Example: samples/BikeSharingApp/Bikes. 
+# <Path to the Dockerfile>                - Path to the folder which contains the Dockerfile
+# <Name of the Service>                   - Pass this as a parameter for multiple steps below. Lower Cased Name of the Service, for example: bikes
+# <Path to the Charts for that service>   - Pass this in as a parameter for the Helm Install PR step
+#                                           This is the path to the folder where the charts are present for that service. example: samples/BikeSharingApp/Bikes/charts/bikes 
+# <ReleaseName>                           - Pass this in as a parameter for the Helm Install PR step
+# 
+# Note: This template assumes that you use Helm. If not, replace the Helm Install PR step with your own step

```

Comment:
Maybe bring this up to the top of the template?

Reply:
Yes. I agree. Updated.